README.md

```diff
@@ -1,2 +1,51 @@
-# reviewapps-deploy-status
-A Github Action that tests the deploy status of a Heroku Review App
+# Review Apps deployment status
+A Github Action that tests the deploy status of a Heroku Review App.
+
+## Usage:
+* Include the action in the workflow
+    ```yaml
+    name: Review App Test
+
+    on:
+        pull_request:
+            branches:
+            - master
+
+    jobs:
+    review-app-test:
+
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v1
+        - name: Run review-app test
+        uses: #TODO: After publishing the action
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            span: 10 # optional in seconds
+            statuses: 200, 302, 403 # optional
+    ```
+
+> Note: Work flow should include `pull_request` event.
+
+* `GITHUB_TOKEN` is required to communicate with Github Deployment API.
+* Input description:  
+    All the inputs are optional. If needed, must be provided from the workflow.
+
+    | Name  | Description  | Default   | 
+    |---|---|---|
+    | span | interval to check the build status  | 10  |
+    | statuses | status_codes to be excluded | 200  |
+
+* Some Review Apps have restricted access. Hence `statuses` can be used to allow restricted status code (eg: 403).
+
+
+### TODO:
+- [x] Add docker file.
+- [x] Add action.yml
+- [x] Add the python script.
+- [x] Test with a repository.
+- [x] Add tests.
+- [x] Add docs.
+- [ ] publish the action.
```

Comment:
Add a section on how to install development environment and run tests. 

Can we have CircleCI too?

Reply:
CircleCI added. Will add local development section in another PR (Along with some other minor changes).